src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Form.js

```diff
@@ -9,25 +9,28 @@ import formStyles from './form.scss';
 
 type Props = {
     store: FormStore,
-    onSubmit: () => void,
+    onSubmit: (options: ?Object) => void,
 };
 
 @observer
 export default class Form extends React.Component<Props> {
+    options: ?Object; // TODO: Change this to a better solution
```

Comment:
Please don't forget to fix that right after this PR is merged.

Reply:
Sure